### ISSUE TITLE
Error message not showing up.

### ISSUE THREAD

**Issue by EEO95** [stephsharp/MaterialTextField#22]
Hi,

I'm using your library on the app I'm currently developing. I really liked it, but I've found that the error message is not getting displayed after setting the NSError to the MFTextField object. 

As you can see in the GIF, the line and placeholder color is getting updated, but the message label is not showing up. I'm handling the text changes with delegation (UITextFieldDelegate).

By the way, my app is being written in Swift 4.2, but I'm making sure to pass NSObjects to the MFTextField.

Best regards, and I'll hope for your answer.

![animated](https://user-images.githubusercontent.com/16014335/49549071-ea0f5b00-f8a4-11e8-81fc-2f2e0a054bf3.GIF)

**Comment by stephsharp** [comment]
Hi @EEO95, thanks for reporting this bug. Can I confirm you're using v1.0.0 of MaterialTextField? I made some major updates to the Auto Layout in that version so it's possible something's not quite right. If you're not on the latest version, would you mind updating and letting me know if you're still not seeing the error message? Thanks!

[This could also be something to try](https://github.com/stephsharp/MaterialTextField/issues/6#issuecomment-196645874) if you're using dynamic height tableview cells.

**Comment by EEO95** [comment]
Hello, Stephanie,

I checked the Podfile.lock file and, indeed, I got the v0.2.1. I tried to update this pod, but I cannot get a higher version for some reason. Maybe it's a repository issue. 

Note: I've tried with both "pod 'MaterialTextField'" and "pod 'MaterialTextField', '~> 0.2'", getting the same result. If I'm doing something wrong, please tell me.

**Comment by stephsharp** [comment]
Try `pod 'MaterialTextField', '~> 1.0'`

**Comment by EEO95** [comment]
I figured out it was a deployment target issue (my Podfile's target was, for some reason, 9.0, and the project's 11.0). Once I corrected that line, it finally worked, as v1.0 required a higher minimum deployment target. I still getting the same behavior (the error message shows up when I start typing in another text field). I'll check my implementation, maybe I'm doing something wrong.

Thanks a lot for your help, and rest assured I'll mention your library in my app's credits. Regards!
